bootstrap-obvious-ci-and-miniconda.py

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env python
```

Comment:
@ocefpaf - I'm trying to catch up with some notifications here, and have no idea if there is more context to this conversation (if there is, I'm sorry), but I'd just like to ask that you try to reduce the bluntness of comments like this. I fully approve of directness (i.e. we don't need to beat around the bush), but it is just as easy to be direct *and polite*...

> Why did you removed this!!!!! This is need by many outside of conda-forge!!!!

Could just have easily been written:

> This change will break things for users outside of conda-forge. Would you mind adding it back please?

---------

I understand frustration when on change results in breakage elsewhere, but in general the world is a nicer place when we tone down the aggression (take that as a political statement if you like 😜 ) . No response needed here, I just want to raise the matter for you to consider in future comments.

Thank you.